zerver/tests/test_users.py

```diff
@@ -296,31 +296,33 @@ def test_valid_user_id(self) -> None:
 
         # Invalid user ID
         invalid_uid = 1000
-        self.assertEqual(check_valid_user_id(realm.id, invalid_uid),
+        self.assertEqual(check_valid_user_ids(realm.id, invalid_uid),
```

Comment:
This test is to check error thrown when list is not proper, but mypy throws error of invalid argument type.

Reply:
Check out how we do this in the validator tests in `test_decorator.py`; we use a type annotation for this.